superset-frontend/src/components/ErrorMessage/ErrorMessageWithStackTrace.tsx

```diff
@@ -51,9 +55,9 @@ export default function ErrorMessageWithStackTrace({
   return (
     <ErrorAlert
       level="warning"
-      title={t('Unexpected Error')}
-      subtitle={message}
-      copyText={message}
+      title={title}
+      subtitle={subtitle}
+      copyText={copyText}
```

Comment:
this will have broken the copyText for a bunch of errors above. Can you pass in the message as the copyText in other errors now as well?

Reply:
oh good catch!